form.js

```diff
@@ -0,0 +1,26 @@
+function validation() {
+  const form = document.getElementById('form');
+  const name = form.elements.name.value;
+  const email = form.elements.email.value;
+  const comment = form.elements.comment.value;
+  const message = document.getElementById('message');
+  message.innerHTML = '';
+
+  form.addEventListener('submit', (event) => {
```

Comment:
- [ ] OPTIONAL
The form is accepting `numbers` as inputs for both _name_ and _message_ columns. Kindly add some validations to check on such entries